### PR TITLE
Updated list of contributors and .mailmap file

### PR DESCRIPTION
**Pull request by astrofrog**
### Description

Forward-ported changes from the release branch - we can keep this open and merge once the final v5.0 is released as there may be more contributors by then.

If anyone has time to try and chase down the names of the users that are identified by usernames, please let me know!

Close #12695

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Do the proposed changes actually accomplish desired goals?
- [x] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [x] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [x] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [x] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [x] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [x] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [x] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [x] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: docs/credits.rst [hunk 1]
```diff
@@ -62,18 +67,22 @@ Core Package Contributors
 * Benjamin Winkel
 * Bernardo Sulzbach
 * Bernie Simon
+* Bhavya Khandelwal
+* bhavyakh
```
Comment (pllim) [inline docs/credits.rst:71]: Are you sure these are not the same person?

File: docs/credits.rst [hunk 2]
```diff
@@ -109,28 +120,35 @@ Core Package Contributors
 * Derek Homeier
 * Devin Crichton
 * Diego Alonso
+* diegoasterio
```
Comment (astrofrog) [inline docs/credits.rst:123]: Diego Asterio de Zaballa

File: docs/credits.rst (hunk 1 above)
Comment (astrofrog) [reply docs/credits.rst]: Fixed

File: .mailmap [hunk 3]
```diff
@@ -91,6 +98,7 @@ Eric Koch                    <koch.eric.w@gmail.com> <koch.eric.w@gmail.com>
 E. Madison Bray              <erik.m.bray@gmail.com>
 E. Madison Bray              <erik.m.bray@gmail.com> <embray@stsci.edu>
 E. Madison Bray              <erik.m.bray@gmail.com> <erik.bray@lri.fr>
+E. Rykoff                    <erykoff@stanford.edu>
```
Comment (pllim) [inline .mailmap:101]: Does this need a rebase? I don't see the `\.` thingy that Simon recommended in the other PR.

Comment (astrofrog) [reply .mailmap:101]: I didn't add the escapes to the mailmap file, just to the generated sphinx docs. Whether we include the escapes here should maybe be discussed separately as the mailmap can be used to make other lists where the \ might not be needed.

**Comment by pllim** [comment]
RTD build failed earlier and I accidentally hit the new rebuild option and now it is taking forever. But the CI reporting suggests that you might want to rebase anyway. Can you please rebase?

**Comment by pllim** [comment]
@astrofrog , you need to rebase.

**Comment by astrofrog** [comment]
Ok sounds good I'll try and get to it next week

**Comment by pllim** [comment]
@astrofrog , I rebased for you. FYI

**Comment by pllim** [comment]
No need for full CI. I'll merge when RTD passes.

**Comment by astrofrog** [comment]
Thanks for wrapping this up!

**Pull request merged**